src/Compilers/CSharp/Portable/Binder/Binder_Patterns.cs

```diff
@@ -1114,7 +1111,7 @@ void addSubpatternsForTuple(ImmutableArray<TypeWithAnnotations> elementTypes)
         {
             // TODO: consider refactoring out common code with BindObjectInitializerMember
             BoundImplicitReceiver implicitReceiver = new BoundImplicitReceiver(memberName, inputType);
-            string name = memberName.Identifier.ValueText;
+            string name = memberName.Identifier.ValueText ?? "";
```

Comment:
>.ValueText ?? "" [](start = 47, length = 16)

If you sync to latest `master`, this should no longer be necessary. `ValueText` now returns a not-null value.